rules/static_test.go

```diff
@@ -502,10 +501,6 @@ var _ = Describe("Static", func() {
 				Expect(findChain(rr.StaticFilterTableChains(ipVersion), "cali-FORWARD")).To(Equal(&Chain{
```

Comment:
We are indeed.  If it's OK with you, I'll note that as a todo for testing, and still merge this now?